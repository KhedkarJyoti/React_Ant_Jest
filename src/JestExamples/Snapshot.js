import { Flex } from "antd";
import React from "react";

const Snapshot = ()=>{
    return(
        <Flex gap={30} align='center' vertical>
            <div style={{padding:100, background:'aliceblue'}}>Hello Snapshot...</div>
        </Flex>
    )
}

export default Snapshot;