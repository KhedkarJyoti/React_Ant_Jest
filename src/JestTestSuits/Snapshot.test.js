import React from 'react'
import {cleanup, render} from '@testing-library/react'
import Snapshot from '../JestExamples/Snapshot';
 
afterEach(cleanup);

it('should take a snapshot', () => {
    const { asFragment } = render(<Snapshot />)
    
    expect(asFragment(<Snapshot />)).toMatchSnapshot()
})