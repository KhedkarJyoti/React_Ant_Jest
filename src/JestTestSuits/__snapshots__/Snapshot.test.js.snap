// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="ant-flex css-dev-only-do-not-override-1ae8k9u ant-flex-align-center ant-flex-vertical"
    style="gap: 30px;"
  >
    <div
      style="padding: 100px; background: aliceblue;"
    >
      Hello Snapshot...
    </div>
  </div>
</DocumentFragment>
`;
